jest.config.ts

```diff
@@ -91,6 +93,9 @@ const config: Config = {
   moduleNameMapper: {
     // Ensures style imports don't break tests
     "\\.(css|scss)$": "<rootDir>/test/jest/__mocks__/styleMock.ts",
+    "react-markdown": "<rootDir>/test/jest/__mocks__/react-markdown.tsx",
+    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$":
+      "<rootDir>/test/jest/__mocks__/fileMock.ts",
```

Comment:
**note:** More mocks for imported node modules that aren't transpiled and other file types. Jest isn't running Webpack so it doesn't know what to do when it encounters weird code.